tests/test_segy.py

```diff
@@ -557,8 +471,9 @@ def test_segyio_types(self):
             self.assertIsInstance(f.tracecount, int)
             self.assertIsInstance(f.samples, int)
 
-            self.assertIsInstance(f.depth_plane, DepthPlane)
-            self.assertIsInstance(f.depth_plane[1], np.ndarray)
+            self.assertIsInstance(f.depth_slice, Line)
```

Comment:
Is this an interesting test, and not just a by-product of internal design decisions? Doesn't make all that sense to me to verify that something is of type X, instead of just checking its behaviour.